src/components/SpellDetails.js

```diff
@@ -1,39 +1,70 @@
 import React, { Fragment } from 'react';
 import { useParams } from 'react-router-dom'
+import ClassFilter from './ClassFilter';
 import '../css/SpellDetails.css';
+import Bookmark from './Bookmark';
 import useFetch from '../hooks/useFetch';
 
 const SpellDetails = ({}) => {
   const spellIndex = useParams().index;
   const { data: details, isLoading, error } = useFetch(`https://www.dnd5eapi.co/api/spells/${spellIndex}`)
-    
+  let spell;
+
+  if (!isLoading) {
+   spell = {"index": details.index, "name": details.name, "url": `https://www.dnd5eapi.co/api/spells/${spellIndex}`}
+  }
+
```

Comment:
Vewwwy Nice!  I think this is a great solution to the problem around adding a bookmark from the details page!  We already have all of the data we need, might as well just make our own object to pass to the bookmark component 👍